channel/consensus_channel/consensus_channel.go

```diff
@@ -107,6 +113,24 @@ func (c *ConsensusChannel) Includes(g Guarantee) bool {
 	return c.current.Outcome.includes(g)
 }
 
+// IncludesTarget returns whether or not the consensus state includes a guarantee targeting the given channel
+func (c *ConsensusChannel) IncludesTarget(target types.Destination) bool {
+	return c.current.Outcome.includesTarget(target)
+}
+
+// RemoveProposedFor returns whether or not a proposal exists to remove the guaranatee for the target
+func (c *ConsensusChannel) RemoveProposedFor(target types.Destination) bool {
```

Comment:
Nit: `RemoveProposedFor` sounds like _imperative_ code. I would prefer something like `HasRemovalBeenProposedFor`.

Reply:
Renamed in https://github.com/statechannels/go-nitro/pull/540/commits/53c0a077a286bc8dbf2655fde0d77a4f56a741c0